src/clr-addons/quick-list/quick-list.component.scss

```diff
@@ -0,0 +1,7 @@
+.pl-0-75 {
```

Comment:
Use named classes here please, don't use the "namespace" of third party libraries.
With nesting you should only need one class.